plugins/exaile-shuffle-collection/__init__.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python
+
+# Copyright (C) 2012 Rainer Hihn ( rainer@hihn.org )
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 1, or (at your option)
+# any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+import glib
+import gtk
```

Comment:
Actually, you can remove the 'import glib' and 'import gtk' lines, as you don't use either of those imports in your plugin.